npm-shrinkwrap.json

```diff
@@ -13,12 +13,12 @@
       "resolved": "https://registry.npmjs.org/bignumber.js/-/bignumber.js-1.5.0.tgz"
     },
     "bluebird": {
-      "version": "2.9.12",
-      "resolved": "https://registry.npmjs.org/bluebird/-/bluebird-2.9.12.tgz"
+      "version": "2.9.24",
```

Comment:
Wasn't this specifically to bump bluebird? That would be helpful when parsing history